contents/naturalLanguage/nl.md

```diff
@@ -0,0 +1,195 @@
+<frontmatter>
+  title: Introduction to Natural Language Processing (NLP)
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Natural Language Processing (NLP)
+Authors: [Lum Ka Fai Jeffry](https://github.com/j-lum)
+
+* [What is NLP?](#what-is-nlp)
```

Comment:
Can wrap this TOC in a 
```
<box id="article-toc">`
... 
</box>
```
, this will ensure that the TOC is responsive with the screen re-sizing